Add strictSSL field for npm (#349)
saucelabs/sauce-docs: docs/testrunner-toolkit/configuration/common-syntax.md

    "@cypress/react": "^5.0.1"
```

### `strictSSL`

> __Description__: Whether or not to do SSL key validation when making requests to the registry via https. If this value is not set, npm will set it to `true` by default.

Reviewer: ```suggestion
__Description__: Instructs npm to perform SSL key validation when making requests to the registry via https (`true`) or not (`false`). Defaults to `true` when not set.
```

Author: Thanks!
